src/react-apps/applications/runtime/src/App.tsx

```diff
@@ -11,23 +11,60 @@ import LanguageActions from './shared/resources/language/languageActions';
 import PartyActions from './shared/resources/party/partyActions';
 import ProfileActions from './shared/resources/profile/profileActions';
 import TextResourcesActions from './shared/resources/textResources/actions';
-
+import { get } from './utils/networking';
 import {
+  getEnvironmentLoginUrl,
   languageUrl,
   profileApiUrl,
+  refreshJwtTokenUrl,
 } from './utils/urlHelper';
 
 const theme = createMuiTheme(AltinnAppTheme);
 
+const ONE_MINUTE_IN_MILLISECONDS: number = 60000;
+
 export default function() {
+  let lastRefreshTokenTimestamp: number = 0;
+
+  function setUpEventListeners() {
+    window.addEventListener('mousemove', refreshJwtToken);
+    window.addEventListener('scroll', refreshJwtToken);
+    window.addEventListener('onfocus', refreshJwtToken);
+  }
+
+  function removeEventListeners() {
+    window.removeEventListener('mousemove', refreshJwtToken);
```

Comment:
I have concerns when it comes to performance using this method. This will be triggered a lot of times. Do you have any input here @matsgm? 
![image](https://user-images.githubusercontent.com/10416981/65409089-23d73100-dde7-11e9-8477-4116dcbd0694.png)

Reply:
![Skjermbilde 2019-09-23 kl  09 56 03](https://user-images.githubusercontent.com/10921322/65409606-677e6a80-dde8-11e9-891b-482faee8b159.png)
CPU usage jumps from 0.9% to 3% on my macbook, when using these events.

Reply:
@jezpoz this is assuring. The solution still seems a little "hacky" to me, but maybe this is the only viable solution.

Reply:
This PR will be merged as it is since it contains a backend bugfix. This performance issue will be continued in #2621.